src/server/frontend_wayland/unique_resource_tracker.h

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
You don't actually need the `UniqueResourceTracker` changes at all now, do you?

Reply:
I thought of leaving it separated just in case. But  I agree that it would be easier to review without these changes (and they're easy to replicate). Should be fixed in the latest commit.